### PR TITLE
Add custom video source mechanism

### PR DESCRIPTION
**Pull request by mustafaboleken**
https://github.com/ant-media/Ant-Media-Server/issues/4569

### CODE REVIEW

File: src/main/webapp/js/external/selfie-segmentation/selfie_segmentation.js.1 [hunk 1]
```diff
@@ -0,0 +1,75 @@
+(function(){/*
```
Comment (SelimEmre) [inline src/main/webapp/js/external/selfie-segmentation/selfie_segmentation.js.1:1]: Is this file necessary?

Comment (mustafaboleken) [reply src/main/webapp/js/external/selfie-segmentation/selfie_segmentation.js.1:1]: There is a mistake, when I downloading files. Let me remove it.

**Review by burak-58 — APPROVED** [review]
